README.md

```diff
@@ -160,4 +160,6 @@ I would like to thank Agaba JohnMark for helping finish this project
 
 ## 📝 License <a name="MIT"> MIT</a>
 
+
 This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] This link is broken at the moment, so sorry that it was not pointed out in previous milestones, kindly create a LICENSE file in the root folder and perhaps customize this [MIT LICENSE template](https://gist.github.com/adamveld12/b64e74d14d7d9043b85c) 😄.